scripts/generator/templates/template-release-notes-maintenance.md

```diff
@@ -25,9 +25,9 @@ Here are the highlighted new and improved features in this maintenance release.
 
 # Getting Started with Vaadin 14
 ## App starters
-The best way to get started with Vaadin 14 is to go to [https://vaadin.com/start](https://vaadin.com/start) and pick an app template for the technology stack you’re interested in. 
+The best way to get started with Vaadin 14 is to go to [https://vaadin.com/start](https://vaadin.com/start) and pick an app template for the technology stack you’re interested in.
 
-For the full list of how to get started go to the [GitHub releases](https://github.com/vaadin/platform/releases/tag/14.1.0).
+For the full list of how to get started go to the [GitHub releases](https://github.com/vaadin/platform/releases/tag/14.2.0).
```

Comment:
This should be 14.3.0. We have a typo in 14.2. This file is for maintenance releases and it's used after 0-version is out